testing/libspatialite/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=libspatialite
+pkgver=4.3.0
+_pkgver="$pkgver"a
```

Comment:
Can we use `pkgver="4.3.0a"` instead of using `_pkgver`?

Reply:
What does "a" mean in this case - alpha or like in openssl?

Reply:
I'm not intimately familiar with the project or the naming convention, but I do not think it's `alpha` as they have explicitly used `beta` before. Regardless, if `pkgver="4.3.0a"` works and is acceptable, I'll just do that. I was likely thinking that `pkgver` didn't accept characters, based on the issue I ran into with `testing/nitro` with `dev` being part of the version number.